articles/api-auth/config/using-the-auth0-dashboard.md

```diff
@@ -0,0 +1,27 @@
+# API Authorization: Using the Auth0 Dashboard
+<%=include('../_preview-warning') %>
+
+1. Open the Auth0 Dashboard and browse to the Clients section.
+
+2. Here you will create a new client of type Non Interactive for each of the applications that will consume the API you want to generate access tokens for.
+
+3. Navigate to the API section and create a new "API" by entering a friendly name and identifier for it (ideally this would be the public endpoint of API). This API will represent your resource server.
+The selection of the Signing Algorithm will dictate how the API will validate the access tokens it receives
+  * HS256 (symmetric): signed using the resource server's signing secret
+  * RS256 (asymmetric): signed using Auth0's private key for your account. Verification is done using the corresponding public key, which can be found at the following standard [JWKS (JSON Web Key set)](https://self-issued.info/docs/draft-ietf-jose-json-web-key.html) URL: https://${account.namespace}/.well-known/jwks.json
+
+![](/media/articles/api-auth/apis-create.png)
```

Comment:
To have the ordered list numbering continue after an image, indent the image tags 2 spaces.